src/pybind/mgr/dashboard/frontend/e2e/dashboard.po.ts

```diff
@@ -0,0 +1,63 @@
+import { $, by, element } from 'protractor';
+import { PageHelper } from './page-helper.po';
+
+export class DashboardPageHelper extends PageHelper {
+  pages = {
+    index: '/#/dashboard'
+  };
+
+  checkGroupTitles(index, name) {
+    // Checks that the titles of all the groups on the dashboard are correct
+    const titles = element.all(by.className('info-group-title'));
+    titles
+      .get(index)
+      .getText()
+      .then((txt) => {
+        expect(txt === name);
+      });
+  }
+
+  cellFromGroup(cardName) {
+    // Grabs cell from dashboard page based off the title. Then returns the card
+    // element
+    const str1 = 'cd-info-card[cardtitle=';
+    const str2 = ']';
+    const str3 = str1.concat(cardName);
+    const getCell = str3.concat(str2);
+    return $(getCell);
```

Comment:
Just a suggestion, you can use string literals to compose strings
```suggestion
    return $(`cd-info-card[cardtitle=${cardName}]`);
```